Makefile

```diff
@@ -89,7 +89,10 @@ clippy-fix:
 	$(cargo) +$(nightly) clippy --fix -Z unstable-options --all-targets --allow-dirty --allow-staged
 
 install: tendermint
-	ANOMA_DEV=false $(cargo) install --path ./apps
+	$(cargo) build --release \
+		--manifest-path matchmaker/mm_token_exch/Cargo.toml && \
+	cp target/release/libmm_token_exch.{dll,dylib,so} ${HOME}/.cargo/bin
```

Comment:
This is a shared object, it should belong in some kind of lib directory

Reply:
install(1) is more idiomatic in a makefile install target than cp(1) for various reasons

Reply:
yeah, I was thinking about putting it into `~/.cargo/lib` instead, but cargo doesn't use any such dir atm

Reply:
I'll switch to `install`, thanks!

Reply:
I imagine there would be a ~/.cargo/lib if the rust-lang/cargo#8294 you mentioned were supported; we could consider just unilaterally making it (`install -d`). It won't be in anyone's LD_LIBRARY_PATH, though we locate it with a command line argument anyway so this doesn't matter.

Reply:
I switched it to the `~/.cargo/lib` dir and added additional attempt in the matchmaker runner to try to load from that dir, the current directory or directly from the given path when it's absolute. I've also made the file extension optional to make it more flexible (previously, it had to be specified without extension)